### PR TITLE
pybind/cephfs: fix custom exception raised by cephfs.pyx

### PR DESCRIPTION
**Pull request by ajarr**
Fixes: https://tracker.ceph.com/issues/46360

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/pybind/cephfs/cephfs.pyx [hunk 1]
```diff
@@ -369,7 +369,7 @@ cdef make_ex(ret, msg):
     if ret in errno_to_exception:
         return errno_to_exception[ret](ret, msg)
     else:
-        return Error(msg + ': {} [Errno {:d}]'.format(os.strerror(ret), ret))
+        return OSError(ret, msg)
```
Comment (ajarr) [inline src/pybind/cephfs/cephfs.pyx:372]: This is also similar to what rbd.pyx does,
https://github.com/ceph/ceph/blob/master/src/pybind/rbd/rbd.pyx#L1007

**Comment by batrick** [comment]
jenkins test dashboard backend

**Comment by ajarr** [comment]
@rishabh-d-dave @varshar16 hey! this change should be OK for cephfs-shell as well?

**Comment by varshar16** [comment]
> @rishabh-d-dave @varshar16 hey! this change should be OK for cephfs-shell as well?

Yes, it is fine for cephfs-shell too.

**Pull request merged**